src/Enterspeed.Source.UmbracoCms.Shared.NetCore/Enterspeed.Source.UmbracoCms.Shared/EnterspeedPropertyValidation/EnterspeedPropertyValueValidator.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Enterspeed.Source.Sdk.Api.Models.Properties;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Enterspeed.Source.UmbracoCms.Shared.EnterspeedPropertyValidation
+{
+    public class EnterspeedPropertyValueValidator : IEnterspeedPropertyValueValidator
+    {
+        private bool Valid(IEnterspeedProperty property)
```

Comment:
Just a minor thing, but I like to typically have public methods at the top because that's what you typically would like to see when you go into a class and then you can scroll down to the private methods if you want to see the more detailed implementation :-)